packages/playground/src/components/node_details.vue

```diff
@@ -70,6 +77,7 @@ import { type PropType, ref, watch } from "vue";
 import { useRoute } from "vue-router";
 
 import CountryDetailsCard from "@/components/node_details_cards/country_details_card.vue";
+import cpuBenchmarkCard from "@/components/node_details_cards/cpu_benchmark.vue";
```

Comment:
```suggestion
import CPUBenchmarkCard from "@/components/node_details_cards/cpu_benchmark_card.vue";
```

Reply:
I didn't name it that way because if renamed to this, it would be used in the template as <c-p-u-benchmark />